src/middlewared/middlewared/plugins/support.py

```diff
@@ -169,14 +169,36 @@ async def fetch_categories(self, token):
 
         return data
 
+    @accepts(Password('token'), Str('query'), roles=['SUPPORT_READ'])
+    @returns(List('similar_issues', items=[Dict(
+        'similar_issue',
+        Str('url'),
+        Str('summary'),
+        additional_attrs=True,
+    )]))
+    async def similar_issues(self, token, query):
+        await self.middleware.call('network.general.will_perform_activity', 'support')
```

Comment:
Same here as well, should probably raise a ValidationError if network activity is blocked.